packages/local_auth/local_auth/lib/local_auth.dart

```diff
@@ -1,145 +1,18 @@
-  /// authentication (e.g. lack of relevant hardware). This might throw
-  /// [PlatformException] with error code [otherOperatingSystem] on the iOS
-  /// simulator.
-  Future<bool> authenticate({
-    required String localizedReason,
-    bool useErrorDialogs = true,
-    bool stickyAuth = false,
-    AndroidAuthMessages androidAuthStrings = const AndroidAuthMessages(),
-    IOSAuthMessages iOSAuthStrings = const IOSAuthMessages(),
-    bool sensitiveTransaction = true,
-    bool biometricOnly = false,
-  }) {
-    return LocalAuthPlatform.instance.authenticate(
-      localizedReason: localizedReason,
-      authMessages: <AuthMessages>[iOSAuthStrings, androidAuthStrings],
-      options: AuthenticationOptions(
-        useErrorDialogs: useErrorDialogs,
-        stickyAuth: stickyAuth,
-        sensitiveTransaction: sensitiveTransaction,
-        biometricOnly: biometricOnly,
-      ),
-    );
-  }
-
-  /// Returns true if auth was cancelled successfully.
-  /// This api only works for Android.
-  /// Returns false if there was some error or no auth in progress.
-  ///
-  /// Returns [Future] bool true or false:
-  Future<bool> stopAuthentication() async {
-    if (_platform.isAndroid) {
-      return LocalAuthPlatform.instance.stopAuthentication();
-    }
-    return true;
-  }
-
-  /// Returns true if device is capable of checking biometrics
-  ///
-  /// Returns a [Future] bool true or false:
-  Future<bool> get canCheckBiometrics =>
-      LocalAuthPlatform.instance.deviceSupportsBiometrics();
-
-  /// Returns true if device is capable of checking biometrics or is able to
-  /// fail over to device credentials.
-  ///
-  /// Returns a [Future] bool true or false:
-  Future<bool> isDeviceSupported() async =>
-      LocalAuthPlatform.instance.isDeviceSupported();
-
-  /// Returns a list of enrolled biometrics
-  ///
-  /// Returns a [Future] List<BiometricType> with the following possibilities:
-  /// - BiometricType.face
-  /// - BiometricType.fingerprint
-  /// - BiometricType.iris (not yet implemented)
-  Future<List<BiometricType>> getAvailableBiometrics() =>
-      LocalAuthPlatform.instance.getEnrolledBiometrics();
-}
+export 'package:local_auth/src/local_auth.dart' show LocalAuthentication;
+
+// TODO(BeMacized): This should be removed the next time a breaking change
+// occurs and LocalAuth#authenticate is removed. Packages will be expected
+// to depend on the platform specific packages directly if they wish to use
+// these classes.
+export 'package:local_auth_android/types/auth_messages_android.dart'
+    show AndroidAuthMessages;
+// TODO(BeMacized): This should be removed the next time a breaking change
+// occurs and LocalAuth#authenticate is removed. Packages will be expected
+// to depend on the platform specific packages directly if they wish to use
+// these classes.
+export 'package:local_auth_ios/types/auth_messages_ios.dart'
+    show IOSAuthMessages;
```

Comment:
@stuartmorgan As we're [bumping the major version](https://github.com/flutter/plugins/pull/5195#discussion_r845288546) here, should we already remove `authenticate` here without deprecating it first?